queue_job/readme/HISTORY.rst

```diff
@@ -11,14 +11,8 @@
 Next
 ~~~~
 
-* [IMP] Dont' start the Jobrunner if root channel's capacity
```

Comment:
I'm curious, do we really want to remove previous version history?

Reply:
Not sure, but as anyway the history is very partial, I'm not sure it's useful. As the branching point is indicated, it's maybe better to look in the 12.0 branch if we want it.